ansible/playbook.yaml

```diff
@@ -0,0 +1,49 @@
+---
+- name: orchestrate components
+  hosts: all
+  vars:
+    favcolor: blue
+  vars_files:
+    - "{{ playbook_dir }}/vars/components.yaml"
```

Comment:
Considering all things are prepared before runtime (not really deploy on those hosts). It seems we can not use `host_vars` and `group_vars` for `hadoop-master1` and `hadoop-worker(1-3)` when building `Dockerfile` for them. Correct me, if I am wrong.